src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -70,6 +72,46 @@
   @JsonProperty("Networks")
   public abstract ImmutableMap<String, AttachedNetwork> networks();
 
+  @Nullable
+  @JsonProperty("EndpointID")
+  public abstract String endpointId();
+
+  @Nullable
+  @JsonProperty("SandboxID")
+  public abstract String sandboxId();
+
+  @Nullable
+  @JsonProperty("SandboxKey")
+  public abstract String sandboxKey();
+
+  @Nullable
+  @JsonProperty("HairpinMode")
+  public abstract Boolean hairpinMode();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6Address")
+  public abstract String linkLocalIpv6Address();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6PrefixLen")
+  public abstract Integer linkLocalIpv6PrefixLen();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6Address")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract String globalIpv6Address();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6PrefixLen")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract Integer globalIpv6PrefixLen();
```

Comment:
UPDATE: I'm trying to go all-`Ip`-everywhere. But I can't do that, at least not if I leave the existing methods there but `@Deprecate` them. If I leave things in the state with two methods, for example, named `fooIpv6` and `fooIPv6`, FindBugs doesn't like it. And not just the two methods themselves—that I can fix with an `@SuppressFBWarnings` annotation.

The problem is that FindBugs sees one method in the AutoValue generated class and the similarly-named abstract method in the parent class and declares them "confusing." The reason I can't solve this with annotations is that AutoValue.Builder classes don't inherit annotations from parent methods (google/auto#409). 

So, while I want to go all-`Ip`, I either can't do it and I should just change my new methods to `IP`, or I can do it but I have to make a breaking change by removing the `IP` methods.